web/forms/contact_form.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Community.ContactForm do
+  use Community.Web, :model
+
+  embedded_schema do
+    field :body, :string
+    field :email, :string
+    field :subject, :string
+    field :member
```

Comment:
I think you could omit the `, :string` or add `, :string` to `:member`. Otherwise it seems like member is something other than a string